### DISCUSSION TITLE
Recommending boosted BLAS

### DISCUSSION THREAD

**Discussion by Polkas** [amices/mice#360] [Ideas]
As a author of the code You should be aware that you mainly using a closed equation solution for stat models. This decision brings better performance compared to gradient decent only when user have a decent algebra library. Nowadays iOSmac users have already a optimized linear algebra libraries. However Windows and Linux users have to download them by their own.
I added some hint in my README:

```
Recommended to download boosted BLAS library:

Windows Users recommended to download MRO MKL: https://mran.microsoft.com/download

Linux users recommended to download Optimized BLAS (linear algebra) library: sudo apt-get install libopenblas-dev
```

We know that this decision brings a huge performance gains with almost no additional effort.

PS. MRO seems not be be updated for at least one year although they announce to provide a version for R 4.0.2

**Comment by stefvanbuuren** [comment]
Thanks for your suggestion. 

I will add a generic note that updating the BLAS can improve speed, and refer users to the discussion in "R Installation and Administration" guide for further details.

**Comment by stefvanbuuren** [comment]
Commit 03276b5 adds note
